src/components/study-container.js

```diff
@@ -561,31 +551,33 @@ export function StudyContainer({ view, onChangeTab }) {
             ) {
                 // study partial update :
                 // loading equipments involved in the study modification and updating the network
-                const [substationsIds, deletedEquipment] =
+                const [substationsIds, deletedEquipments] =
                     parseStudyNotification(studyUpdatedForce);
-
+                if (deletedEquipments?.length > 0) {
+                    // removing deleted equipment from the network
+                    deletedEquipments.forEach((deletedEquipment) => {
+                        if (deletedEquipment?.id && deletedEquipment?.type) {
+                            console.info(
+                                'removing equipment with id=',
+                                deletedEquipment?.id,
+                                ' and type=',
+                                deletedEquipment?.type,
+                                ' from the network'
+                            );
+                            network.removeEquipment(
+                                deletedEquipment?.type,
+                                deletedEquipment?.id
+                            );
+                        }
+                    });
+                    dispatch(setDeletedEquipments(deletedEquipments));
```

Comment:
Do we use the redux to dispatch the deleted equipments or we catch notifications the same way as here in the nework-map-tab ?

Reply:
To be done later